aws-integrations/firehose/README.md

```diff
@@ -12,11 +12,10 @@ For a more detailed description of the settigns and architecture of this AWS Kin
 
 | Parameter | Description | Default Value | Required |
 |---|---|---|---|
-| CoralogixRegion | The region of your Coralogix Account | _Allowed Values:_<br>- ireland<br>- stockholm<br>- india<br>- singapore<br>- us<br>- us2<br>_Default: ireland_ | :heavy_check_mark: |
+| CoralogixDomain | The Coralogix Url domain. If set, will be the url used to send telemetry (e.g. coralogix.com, eu2.coralogix.com) | | |
```

Comment:
The description of the parameter is perfectly correct, but the way it is used, I think the URL value should be  
`Url: !Sub "https://firehose-ingress.${CustomDomain}/firehose`

Reply:
On our side, the Coralogic Domain is required because we use the same code base for all the CF integrations. It'd be great if we can keep things consistent across all the CF integrations. For other CF integrations we only provide the domain and the url is build by the template, so we don't have to know it on our side

Reply:
i'll make that good on the next PR, roy explained it to me